### ISSUE TITLE
Gas Mask wants to make changes dialog box

### ISSUE THREAD

**Issue by robwired** [2ndalpha/gasmask#156]
The system dialog box appears continuously with, "Gas Mask wants to make changes. Enter your password to allow this." 

I think this is happening because Parallels for Business for Mac is making a change to the hosts file and Gas Mask is correctly changing the hosts file back but needs a super-privilege so that I don't have to manually confirm this change over and over.
![2018-10-13_16-17-33](https://user-images.githubusercontent.com/1443811/46910941-7f285000-cf03-11e8-868f-d3b5bfb4981e.jpeg)

**Comment by robwired** [comment]
Here is the Apple System Notification that appears after I authorize the change.

![2018-10-15_13-08-03](https://user-images.githubusercontent.com/1443811/46975412-62168d00-d07b-11e8-8a92-ea9d45dbba7b.jpeg)

**Comment by JOGUI22** [comment]
@robwired I'm having the same problem, did you fix it?

**Comment by RougeTomate** [comment]
Same thing here. I get this message 15-20 times a day. Please, fix. 

Thanks.

**Comment by stantastick** [comment]
Yep, me too. Of course there's no real reason I NEED to have it running all day. I can just select a host file and then quit the app. But I don't consider it popping up every so often a redeeming quality.

**Comment by essgeejay** [comment]
I found a KB that seems to fix it. [https://kb.parallels.com/en/123300](url)

**Comment by hiarmen** [comment]
This issue became a major irritator convincing me to remove Gas Mask.

**Comment by hiarmen** [comment]
> I found a KB that seems to fix it. [https://kb.parallels.com/en/123300](url)

Open the /Library/Preferences/Parallels/network.desktop.xml location, copy this xml file to desktop and change values from 1 to 0:
NATDAllowEtcHostsWrite: 0
RegVmNameInEtcHosts: 0
Then copy the file back [replace]
No more password popups.
